web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
+	router.Get("/graph", func(w http.ResponseWriter, r *http.Request) {
+		http.Redirect(w, r, "/", http.StatusMovedPermanently)
```

Comment:
Let's keep it the other way around – `/` redirecting to `/graph`. That keeps more freedom to change the root page again later.